analysis/parser/RawTextParser/src/main/kotlin/de/maibornwolff/codecharta/parser/rawtextparser/metrics/Metric.kt

```diff
@@ -0,0 +1,18 @@
+package de.maibornwolff.codecharta.parser.rawtextparser.metrics
+
+import de.maibornwolff.codecharta.parser.rawtextparser.model.FileMetrics
+
+interface Metric {
+
+    val name: String
+    val description: String
+
+    fun setParameters(parameters: Map<String, Int>) {
```

Comment:
are the { } required here?

Reply:
The { } provides a default implementation of the function. If you have a metric that does not require to set parameters, you are not forced to override the function with an empty function.